src/doc/rustdoc/src/lints.md

```diff
@@ -0,0 +1,118 @@
+# Lints
+
+`rustdoc` provides lints to help you writing and testing your documentation. You
+can use them like any other lints by doing this:
+
+```rust
+#![allow(missing_docs)] // allowing the lint, no message
+#![warn(missing_docs)] // warn if there is missing docs
+#![deny(missing_docs)] // rustdoc will fail if there is missing docs
+```
+
+Here is the list of the lints provided by `rustdoc`:
+
+## intra_doc_link_resolution_failure
```

Comment:
Should we label this one "nightly only" ?

Reply:
Also should we say this is warn-by-default?

Reply:
Yes for both!